index.html

```diff
@@ -323,155 +327,186 @@ <h1>Multi-Post Stories Gain+Glory</h1>
+									</a>
 								</li>
 							</ul>
 
 						</div>
-			</div>
+					</div>
 
-			<!-- about-me-card  3 -->
-			<div class="about-me-cards">
-				<div class="icon-languages">
-					<img src="images/icon-skills.svg" alt="languages">
-				</div>
-				<div>
-					<h1>
-					   Skills 
-					</h1>
-				</div>
+					<!-- about-me-card  3 -->
+					<div class="about-me-cards">
+						<div class="icon-languages">
+							<img src="images/icon-skills.svg" alt="languages">
+						</div>
+						<div>
+							<h1>
+								Skills 
+							</h1>
+						</div>
 						<div class="abt-frameworks">
 							<ul class="abt-languages">
 								<li>
 									<a href="#">
-								<button id="ruby">Codekit</button>
-								</a>
+										<button id="ruby">Codekit</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">GitHub</button>
-								</a>
+										<button id="ruby">GitHub</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Codepen</button>
-								</a>
+										<button id="ruby">Codepen</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">GitLab</button>
-								</a>
+										<button id="ruby">GitLab</button>
+									</a>
 								</li>
 								<li>
 									<a href="#">
-								<button id="ruby">Terminal</button>
-								</a>
+										<button id="ruby">Terminal</button>
+									</a>
 								</li>
 							</ul>
 
 						</div>
+					</div>
+
+				</div>
 			</div>
+		</section>
+		<section>
+			<div class="form-section">
+				<div> 
+					<h1 class="frm-txt">I'm always interested in hearing about new projects,
+					so if you'd like to chat please get in touch.</h1>
+				</div>
+				<form action="https://formspree.io/f/xwkzwlne" method="POST" >
+					<input class="form-input" type="text" name="name" placeholder="Full Name" maxlength="30" required>
+					<input class="form-input" type="email" name="email" placeholder="Email Address" required>
+					<textarea class="form-input form-input2" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+					<button  type="submit">Get in touch</button>
+				</form>
+				<div class="form-img">
+					<img src="images/Frame.svg" alt="Frame"/>
+					
+				</div>
+				<nav class="social">
+					<ul id="social-icons">
 
-		</div>
-		</div>
+						<li><a class="fa" href="#" target="_blank"><img src="images/twitter.svg" alt="twitter"/></a></li>
+						<li><a class="fa"  href="#" target="_blank"><img src="images/linkedin-icon.svg" alt="Linkedin"/></a></li>
+						<li><a class="fa"  href="#" target="_blank"><img src="images/medium.svg" alt="medium"/></a></li>
+						<li><a class="fa"  href="#" target="_blank"><img src="images/github.svg" alt="github"/></a></li>
+						<li><a class="fa"  href="#" target="_blank"><img src="images/angellist.svg" alt="angellist"/></a></li>
+					</ul>
+				</nav>
```

Comment:
- Good job wrapping the social links in a `ul` tag, but I suggest you use a `footer` tag instead of a `nav` tag to wrap this section, this would help improve the structure of your page. Please fix this 👍 👍

Reply:
Thank you @mckent05 . This will definitely be a better HTML5 semantic.